introduction.md

```diff
@@ -1,2 +1,7 @@
 Hi my name is diego I am a programmer and I am doing an amazing github course at codeschool.com
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?